pkg/util/sentry/request.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 var HeaderWhiteList = []string{
+	"Origin",
 	"Referer",
```

Comment:
Try to add it to context in the sentry hook.

Reply:
seems it would work with sentry allow list actually after some time, re: https://linear.app/authgear/issue/DEV-1337/track-csrf-errors-in-sentry#comment-be44f87b

Reply:
let's keep the info here instead?

Reply:
Sure.